src/components/molecules/CheckoutForm/checkoutSchema.ts

```diff
@@ -11,7 +12,7 @@ export const checkoutSchema = object({
     .min(1, 'Email address is required')
```

Comment:
min ---> required

Reply:
I checked it and it seems that every field is required, although it can be changed by .optional() method. I'm aware that email can't be less than 5 characters, but I used .min() to provide descriptive error message to user instead of 'Email is invalid' which will be displayed when user type email in wrong format.

Reply:
It worked for delivery and payment fields, but didn't for email where it showed same error message as for invalid email. So I changed it to nonempty() and provided proper error message.